widgetastic_manageiq/__init__.py

```diff
@@ -260,10 +260,13 @@ def get_text_of(self, item_name):
         """
 
         multiple_lines = self.get_item(item_name).text.splitlines()
-        if len(multiple_lines) > 1:
-            return multiple_lines
+        if multiple_lines:
```

Comment:
this looks too sophisticated.
I'd rather turn it into

`
if:
elif:
else:
`
for better readability.

Reply:
Definitely!